static/game/spawner.js

```diff
@@ -87,7 +87,7 @@ export function populateLevel(board, numEnemies) {
     for (let k = 0; k < numPowerUps; k++) {
       if (Math.random() > 0.75) {
         const p = new powerUpTypes[randomInt(powerUpTypes.length)](
-          randomInt(6)
+          1 + randomInt(5)
```

Comment:
So that's where to 0 magnitude was coming from

Reply:
I thought a 0 magnitude powerup would just be like no powerup.